mavis/summary/summary.py

```diff
@@ -219,21 +219,21 @@ def get_pairing_state(current_protocol, current_disease_state, other_protocol, o
     dt = (PROTOCOL.TRANS, DISEASE_STATUS.DISEASED)
     ng = (PROTOCOL.GENOME, DISEASE_STATUS.NORMAL)
 
+    if current_protocol != other_protocol:
+        is_matched = is_matched or inferred_is_matched
+
     if curr == dg and other == ng:
         return PAIRING_STATE.GERMLINE if is_matched else PAIRING_STATE.SOMATIC
     elif curr == dg and other == dt:
-        return PAIRING_STATE.EXP if inferred_is_matched else PAIRING_STATE.NO_EXP
+        return PAIRING_STATE.EXP if is_matched else PAIRING_STATE.NO_EXP
     elif curr == dt and other == dg:
-        return PAIRING_STATE.GENOMIC if inferred_is_matched else PAIRING_STATE.NO_GENOMIC
+        return PAIRING_STATE.GENOMIC if is_matched else PAIRING_STATE.NO_GENOMIC
     elif curr == dt and other == ng:
-        return PAIRING_STATE.GERMLINE if inferred_is_matched else PAIRING_STATE.SOMATIC
+        return PAIRING_STATE.GERMLINE if is_matched else PAIRING_STATE.SOMATIC
     elif curr == ng and other == dt:
-        return PAIRING_STATE.EXP if inferred_is_matched else PAIRING_STATE.NO_EXP
+        return PAIRING_STATE.EXP if is_matched else PAIRING_STATE.NO_EXP
     else:
-        if current_protocol == other_protocol:
-            return PAIRING_STATE.MATCH if is_matched else PAIRING_STATE.NO_MATCH
-        else:
-            return PAIRING_STATE.MATCH if inferred_is_matched else PAIRING_STATE.NO_MATCH
+        return PAIRING_STATE.MATCH if is_matched else PAIRING_STATE.NO_MATCH
```

Comment:
looks like some of these conditions can be merged together to improve clarify

Reply:
not sure why this isn't showing up in the full diff. I made these changes

Reply:
That's weird. I see the change in that commit. But on the main pr page it doesn't show up. anyways, thanks